src/App.js

```diff
@@ -1,9 +1,26 @@
+import { Link, Route, Routes } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import QuoteDisplay from './components/QuoteDisplay';
+import Home from './components/Home';
+import classes from './App.module.css';
 
 function App() {
   return (
     <>
-      <Calculator />
+      <nav className={classes['nav-bar']}>
+        <h1>Math Magician</h1>
+        <ul className={classes['nav-list']}>
+          <li className={classes['nav-link']}><Link to="/">Home</Link></li>
+          <li className={classes['nav-link']}><Link to="/calculator">Calculator</Link></li>
+          <li className={classes['nav-link']}><Link to="/quot">Quote</Link></li>
+        </ul>
+      </nav>
```

Comment:
[OPTIONAL] You can improve the React application by making it more modular by separating the navigation into it's own `Navbar.js` component.

- This will make the Navbar reusable and offers a versatile layout for situations where you need to render different layouts for other pages.